mu4/notation/view/widgets/transposedialog.h

```diff
@@ -0,0 +1,92 @@
+//=============================================================================
+//  MusE Score
+//  Linux Music Score Editor
+//
+//  Copyright (C) 2008-2009 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MU_NOTATION_TRANSPOSEDIALOG_H
+#define MU_NOTATION_TRANSPOSEDIALOG_H
+
+#include "ui_transposedialog.h"
+#include "libmscore/mscore.h"
```

Comment:
Please get rid of this dependency in this file.

Reply:
fixed